File: app/get_new_prompt.py
from prompt_list import journal_prompts
import random
import string

def get_new_journal_prompt(journal_prompts):
    """Returns a new random journal prompt."""
    
    #shuffle prompts list
    random.sample(journal_prompts, len(journal_prompts))
    #TODO: return [0] from list after shuffling
    return journal_prompts[0]

File: app/get_new_prompt.dart
//TODO this isn't working
import prompt_list from journal_prompts;

import 'dart:math';

void get_new_journal_prompt(journal_prompts) {
  //returns random new journal prompt
  // TODO return random object from journal prompt list
  int randomIndex = Random().nextInt(prompt_list.length);
  
  print(prompt_list[randomIndex]);
};